flake.nix

```diff
@@ -123,7 +123,7 @@
         in pkgs.haskell-backend.pkgSet.shellFor {
           packages = pkgs.haskell-backend.localPkgsSelector;
           nativeBuildInputs =
-            [ pkgs.haskell.packages.ghc928.cabal-install pkgs.z3 ];
+            [ pkgs.haskell.packages.ghc928.cabal-install pkgs.z3 pkgs.libffi ];
```

Comment:
do we still need this change??

Reply:
Maybe not - have reverted to see if it was just the cache all along (which I suspect will be the case given the CI passes elsewhere)